test/sql/test_substrait_enable_optimizer.test

```diff
@@ -56,7 +56,7 @@ CALL get_substrait('select * from tbl where v LIKE ''duck%''');
 query I
 CALL get_substrait('select * from tbl where v LIKE ''duck%''', enable_optimizer=true);
 ----
-<REGEX>:.*starts_with:string_string.*
+<REGEX>:.*gte:string_string.*
```

Comment:
Why did we end up needing to make this change?

Reply:
The plan has changed to use `gte:string_string` and `lt:string_string` istead of `starts_with`